metabase/metabase
# Let users know why there’s no drill-through on their SQL/native questions

Epic https://github.com/metabase/metabase/issues/26565
Demo https://www.loom.com/share/aa08637f929049e8975c8c74a77aa589

How to test:
- New -> Question -> Orders, sum Count, group by Created At (month)
- Convert the question to SQL, save it, and add it to a dashboard
- Click on a dot on the chart. It should show the message below. The same should work for this question in the dashboard.
- Share the question and the dashboard. Make sure that clicking on a dot doesn't do anything there, as before.

<img width="601" alt="Screenshot 2022-11-17 at 14 13 11" src="https://user-images.githubusercontent.com/8542534/202443314-34dac615-5eb6-424a-a8a7-c27e814914f3.png">
